templates/default/etc/init.d/opentsdb_debian

```diff
@@ -16,8 +16,8 @@
 
 PATH=/bin:/usr/bin:/sbin:/usr/sbin
 NAME=opentsdb
-TSD_USER=opentsdb
-TSD_GROUP=opentsdb
+TSD_USER=<% @options[:opentsdb_resource].user %>
+TSD_GROUP=<% @options[:opentsdb_resource].group %>
```

Comment:
So if you see here

https://github.com/acaiafa/opentsdb-cookbook/blob/master/libraries/opentsdb_instance.rb#L36-L40

I am calling those directly.